stages/apps/sharedsvc/variables.tf

```diff
@@ -19,12 +22,19 @@ variable "enable_autoscaler_scale_down" {
   default = true
 }
 
+variable "enable_dashboard" {
+  default = true
```

Comment:
I think we wanted to default all the enable flags to false, to make it easier to see what's enabled for a given environment in lead-environments